packages/spelunker-api/src/lib/entities/Item.mjs

```diff
@@ -27,6 +27,12 @@ class Item extends DatabaseEntity {
     query.where('name', 'LIKE', `%${searchQuery}%`);
   }
 
+  static filter(query, filters) {
+    if (filters?.itemClassIds?.length > 0) {
+      query.andWhere('class', 'IN', filters.itemClassIds);
+    }
+  }
```

Comment:
Seeing as this filter is specific to `Item`, perhaps we should drop the `item`-prefix and instead have `classIds` as the filter name? (is shorter, too 😊)

Reply:
I've been waffling on that. Because the game also has a notion of allowed character classes for items as well (eg wands can only be used by casters), I think being very unambiguous for item class is the better choice – even if it's verbose.

Reply:
For consistency, I think we'd stick to that overly verbose naming for `ItemSubClass` as well. But I'm on board with our normal convention of stripping `item` prefixes from related attributes of items (eg. `displayInfo` instead of `itemDisplayInfo`).

Reply:
Ah, very good point on the character class stuff, wasn't aware of that 👍